custom_model_runner/datarobot_drum/drum/drum.py

```diff
@@ -425,6 +436,10 @@ def run(self):
             raise DrumCommonException(error_message)
 
     def run_fit(self):
+        input_data = pd.read_csv(self.options.input)
+        if self.options.target:
+            input_data.drop(self.options.target, inplace=True, axis=1)
+        self.schema_validator.validate_inputs(input_data)
```

Comment:
I wonder if we could validate later on after we've read in the CSV for other purposes.